charts/keda/templates.yaml

```diff
@@ -1 +1,9 @@
-keda: https://github.com/kedacore/keda/raw/fed1348af0c393186c931e482021b10413bad62e/deploy/KedaScaleController.yaml
+namespace: https://storage.googleapis.com/projectriff/charts/extras/keda/keda-namespace.yaml
+crd-scaledobject: https://github.com/kedacore/keda/raw/v1.0.0/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
+crd-triggerauthentications: https://github.com/kedacore/keda/raw/v1.0.0/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
+api-service: https://github.com/kedacore/keda/raw/v1.0.0/deploy/api_service.yaml
+operator: https://github.com/kedacore/keda/raw/v1.0.0/deploy/operator.yaml
```

Comment:
As I don't know much about Helm, stupid question: can those keys clash when this is "included" in a bigger chart?

Reply:
These keys become files within the helm chart's templates directory. It won't clash.

https://github.com/projectriff/charts/blob/68e14c3da164306c2a20412b9043ba54249d5600/charts/package.sh#L24